hc/accounts/tests/test_login.py

```diff
@@ -16,22 +17,55 @@ def test_it_sends_link(self):
 
         form = {"email": "alice@example.org"}
 
-        r = self.client.post("/accounts/login/", form)
-        assert r.status_code == 302
+        response = self.client.post("/accounts/login/", form)
+        assert response.status_code == 302
```

Comment:
other tests use the format `self.assert`. Its good practice to one format of code, to avoid confusion. Please change this to follow the `self.assert` format.